doc/user/environments.xml

```diff
@@ -1133,7 +1143,7 @@ int main() { }
       <para>
 
       Like the &Environment; call that creates a construction environment,
-      the &Clone; method takes &consvar; assignments,
+      the &Clone; method takes &consvar; assignment keyword arguments,
```

Comment:
I think assignments is better.. less python grok'ing needed.

Reply:
I can change back.